src/arguments/update.py

```diff
@@ -0,0 +1,35 @@
+import requests as requests
+from termcolor import colored
+import webbrowser as webbrowser
+
+from .error import SearchError
+
+class UpdateApplication(object):
+    def __init__(self, current_version):
+        """

```

Comment:
Add a blank line b/w summary line and description

Reply:
+1

Reply:
Ohh, fine. Thank you